### PR TITLE
all: update graph-notebook and amazon-neptune-tools

### PR DESCRIPTION
**Pull request by jroimartin**
This PR updates graph-notebook and amazon-neptune-tools to their latest
versions. It also pins all Python dependencies using pip-compile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,38 +10,51 @@ IAM credentials.
 
 ## Using neptune-python-utils
 
-1. Run the docker image passing the environment variables corresponding to your
-   "AWS variables", Neptune's host and Neptune's port:
+Run the docker image passing the environment variables corresponding to your
+"AWS variables", Neptune's host and Neptune's port:
 
-```bash
```
Comment (manelmontilla) [inline README.md:16]: Just curious:
Why are we removing the language here?

**Review by manelmontilla — DISMISSED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (jroimartin) [reply README.md:16]: I removed them because they are not bash script snippets but command-line instructions. Does it make sense?

**Review by manelmontilla — APPROVED** [review]
LGTM

**Pull request merged**